src/commands/install.rs

```diff
@@ -153,6 +175,8 @@ pub enum Error {
     CantFindNodeVersion { requested_version: UserVersion },
     #[error("Can't find relevant LTS named {}", lts_type)]
     CantFindRelevantLts { lts_type: crate::lts::LtsType },
+    #[error("Can't find latest version")]
+    CantFindLatest,
```

Comment:
I wonder if we should be more explicit about the issue, like: "can't find any versions in the upstream version index"

Reply:
resolved, thx for suggestion